core/queryparser/sparql/src/main/java/org/eclipse/rdf4j/query/parser/sparql/SPARQLParser.java

```diff
@@ -130,23 +130,23 @@ public ParsedUpdate parseUpdate(String updateStr, String baseURI) throws Malform
 
 					String datablock = "";
 					if (updateExpr instanceof InsertData) {
-						InsertData insertDataExpr = (InsertData)updateExpr;
+						InsertData insertDataExpr = (InsertData) updateExpr;
 						parser.getParserConfig().set(BasicParserSettings.SKOLEMIZE_ORIGIN, null);
 						parser.setLineNumberOffset(insertDataExpr.getLineNumberOffset());
 						datablock = insertDataExpr.getDataBlock();
 					} else if (updateExpr instanceof DeleteData) {
-						DeleteData deleteDataExpr = (DeleteData)updateExpr;
+						DeleteData deleteDataExpr = (DeleteData) updateExpr;
 						parser.setLineNumberOffset(deleteDataExpr.getLineNumberOffset());
 						parser.setAllowBlankNodes(false);
 						datablock = deleteDataExpr.getDataBlock();
 					}
-					if(!datablock.equals("")){
+
+					if (!datablock.equals("")) {
```

Comment:
Thanks for the help @jeenbroekstra . We have done all the changes suggested above.

We also noted one thing in the SPARQLParser.java file. In case the `updateNode` itself is not null, then do we need to check separately that the `datablock` is empty? We have kept this condition for now.

Reply:
Yeah, I think that how you currently have it is correct: you only want to process the datablock if it is not empty. Even if `updateNode` is not null, it can still be a different type of update, e.g. a  `Modify` operation, in which case there is no datablock to parse. 

I don't think it matters too much because the parser will return very quickly on an empty string, but I think doing a quick string equality check is marginally more efficient.